packages/mds-metrics/tests/test.ts

```diff
@@ -27,11 +26,11 @@ const AUTH_NO_SCOPE = SCOPED_AUTH([], TEST1_PROVIDER_ID)
 const CITY_OF_LA = '1f943d59-ccc9-4d91-b6e2-0c5e771cbc49'
 
 before(async () => {
-  await Promise.all([db.initialize(), cache.initialize(), stream.initialize()])
+  await Promise.all([db.initialize(), cache.initialize()])
```

Comment:
Seems like maybe the tests should exercise both the NATS and KAFKA flows? Or do they do that already and I'm missing it?

Reply:
Unless you'd like to be running Kafka/NATS on every developer machine (much more hefty than PG/Redis), it's extremely hard to do that. I also haven't written any consumer code, so testing I/O will be very non-trivial. We'll need to write _actual_ integration tests at some point that have an actual solid infrastructure behind them to hook into CI/CD.

Reply:
I wonder if we could just mock the relevant kafka API methods we expect to be called and add some assertions around the call parameters.